drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -279,6 +279,11 @@ config NRF700X_RPU_PS_IDLE_TIMEOUT_MS
 	int "RPU power save idle timeout in milliseconds"
 	default 10
 
+config NRF700X_RPU_EXTEND_TWT_SP
+	bool "Enable extending TWT service period"
+	help
+	  Enabling this feature extends TWT service period to send frames.
+	  It uses EDCA to avoid dropping of frames, so consumes more power.
```

Comment:
This needs more clarification to convey that `new frames` aren't send outside SP, only accepted frames before the SP are sent outside SP.

Reply:
Updated help. Please check